package.json

```diff
@@ -1,14 +1,13 @@
 {
   "name": "ngx-mqtt",
   "description": "ngx mqtt client library",
-  "version": "7.0.14",
+  "version": "8.0.0",
```

Comment:
we don't need to update major version if no breaking changes in update.
Library contract is untouched, updating deps is not a reason.
Please see second comment anot peerDeps

Reply:
Giviving package version 7.0.15 instead